scripts/build/libc/glibc.sh

```diff
@@ -352,6 +352,12 @@ do_libc_backend_once() {
             build_cppflags="${build_cppflags} -I${CT_BUILDTOOLS_PREFIX_DIR}/include/"
             build_ldflags="${build_ldflags} -lintl -liconv"
             ;;
+        *freebsd*)
+            # .. and the same goes for FreeBSD. Additionally, FreeBSD's stat is
+            # always 64bit, so replace struct stat64 with stat.
+            build_cppflags="${build_cppflags} -I${CT_BUILDTOOLS_PREFIX_DIR}/include/ -Dstat64=stat"
```

Comment:
Oh wow, and looking at it, that patch has been there for years. I think that since multiple platforms don't have stat64 (and I can see the other BSDs being added to the list) it might be easier to pass in -Dstat64=stat.